docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+    3. Enter the `./dcrwallet --create` command
+
+**Linux**: <br />
+    1. Using anything you like, navigate to the directory of your `dcrwallet` executable <br />
+    2. Enter the `./dcrwallet --create` command
+
+This will run you through the usual prompts of creating a new wallet. Follow the directions on the screen. You will be required to create a private passphrase (you will use this later to unlock your wallet while creating transactions). The additional layer of encryption is completely optional. Your seed can be used to restore your wallet on any computer using dcrwallet. Record your seed, put it in a safe place, and never share it with anyone. The wallet will close once everything is finished.
+
+> Step 5: Start dcrd on the Testnet
+
+Run your `dcrd` executable with the `testnet=1` or `testnet=true` option within the config file to start your node on the testnet. Your node will begin to sync with the rest of the network. Syncing will take awhile.
+
+> Step 6: Start dcrwallet on the Testnet
+
+Run your `dcrwallet` executable with the `testnet=1` or `testnet=true`option within the config file to start your wallet on the testnet. Your wallet will connect to your node and begin to sync your addresses. Syncing may take awhile.
+
+> Step 7: Register at Stakepool Website
+
+While waiting for your node/wallet to sync, visit [https://teststakepool.decred.org](https://teststakepool.decred.org) and register for a new account. Continue to step 8.
+
+> Step 8: Wait for Testnet Node/Wallet to Sync
+
+Take a break, this may take awhile. Your CPU will be taxed heavily during this syncing.
+
+> Step 9: Create a Public Key Address to Use the Stakepool
+
+The first step to using a stakepool is the generate a new public key address to be used to generate a 1-of-2 multisignature script. Follow the directions on [https://teststakepool.decred.org/address](https://teststakepool.decred.org/address) to generate and save this address. If you've set up an account on a mainnet stakepool, this is the same.
+
+> Step 10: Import Your P2SH Multi-Sig Script From the Stakepool
+
+Next, you need import a script that will allow you to delegate voting rights to the pool. After completing the previous step, this script should be available at [https://teststakepool.decred.org/tickets](https://teststakepool.decred.org/tickets). Again follow the directions there to import the script. If you've set up an account on a mainnet stakepool, this is the same.
+
+> Step 11: Acquire Testnet Coins
+
+Next, you'll need to acquire Testnet coins in order to buy Testnet tickets. There is am official Testnet faucet located at [https://faucet.decred.org](https://faucet.decred.org). Enter a Testnet address (one can be retrieved by running the `getnewaddress` command - examples for each OS below)
+
+    Windows: dcrctl.exe --wallet getnewaddress
+    macOS: ./dcrctl --wallet getnewaddress
```

Comment:
Maybe combine the macOS and Linux into one line since the commands are identical.
macOS/Linux: ./dcrctl --wallet getnewaddress

But if you prefer it as is I'm okay with that.